src/EditorFeatures/Core/Implementation/ReferenceHighlighting/AbstractDocumentHighlightsService.cs

```diff
@@ -130,8 +130,10 @@ private bool ShouldConsiderSymbol(ISymbol symbol)
             bool addAllDefinitions = true;
 
             // Add definitions
+            // Filter out definitions that cannot be highlighted. e.g: alias symbols defined via project property pages.
             if (symbol.Kind == SymbolKind.Alias &&
-                symbol.Locations.Length > 0)
+                symbol.Locations.Length > 0 &&
+                symbol.Locations.First().SourceTree != null)
```

Comment:
Ah nice catch! I failed to notice that side-effect flowing through since I get 0 references for now (2079) :smiley: I guess the [`location.IsInSource`](https://github.com/dotnet/roslyn/pull/2085/files#diff-6eed65d33ebfd49a9af243b54de70500R150) check should be sufficient but let me double check and also set it correctly. Thanks.